tests/docker_mock.py

```diff
@@ -229,13 +268,13 @@ def mock_docker(build_should_fail=False,
     :param build_should_fail: True == build() log will contain error
     :param inspect_should_fail: True == inspect_image() will return None
     :param wait_should_fail: True == wait() will return 1 instead of 0
-    :param provided_image_repotags: images() will contain provided image
+    :param provided_image_repotags: images() will contain provolume_nameed image
```

Comment:
Right, I shouldn't have replaced all `vid` instances to `volume_name` blindly. Fixed